.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+# This job runs the project tests
+
+name: run-lusid-python-tools-tests
+
+# Trigger the workflow on push or pull request to master or develop
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Run tests on MASTER branch
+        if: ${{ github.base_ref == 'master' }}
+        env: 
+          FBN_TOKEN_URL: ${{ secrets.MASTER_FBN_TOKEN_URL }}
+          FBN_USERNAME: ${{ secrets.MASTER_FBN_USERNAME }}
+          FBN_PASSWORD: ${{ secrets.MASTER_FBN_PASSWORD }}
+          FBN_CLIENT_ID: ${{ secrets.MASTER_FBN_CLIENT_ID }}
+          FBN_CLIENT_SECRET: ${{ secrets.MASTER_FBN_CLIENT_SECRET }}
+          FBN_LUSID_API_URL: ${{ secrets.MASTER_FBN_LUSID_API_URL }}
+          FBN_APP_NAME: ${{ secrets.MASTER_FBN_CLIENT_ID }}
+        run: | 
+          echo "env variables for MASTER have been set"
+          docker build -t finbourne/lusid-python-tools-test .
+          docker run \
+            -e FBN_TOKEN_URL="$FBN_TOKEN_URL" \
+            -e FBN_USERNAME="$FBN_USERNAME" \
+            -e FBN_PASSWORD="$FBN_PASSWORD" \
+            -e FBN_CLIENT_ID="$FBN_CLIENT_ID" \
+            -e FBN_CLIENT_SECRET="$FBN_CLIENT_SECRET" \
+            -e FBN_LUSID_API_URL="$FBN_LUSID_API_URL" \
+            -e FBN_APP_NAME="$FBN_APP_NAME" \
+            -v $(pwd):/usr/src/ \
+            finbourne/lusid-python-tools-test
+          
+      - name: Run tests on DEVELOP branch
```

Comment:
don't think we need this do we?

Reply:
@slemasne  this is just what I was about to ask you on the call

Reply:
Thanks, done.